src/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,25 @@
+.layout-container {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    /* background-color: #525f7f; */
+}
+
+.content-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+
+main {
+    height: 100%;
+    overflow: hidden;
+    /* padding: 10px; */
+}
+
```

Comment:
Maybe try to avoid styling elements directly in css. As far as I saw most of us avoided that by just implementing something like this  `<main className{styles['main']>{children}</main>`  and then just  ```
.main {
}
``` in css instead of `main { }`